site/content/api/Event-API.md

```diff
@@ -76,4 +81,82 @@ The following sequence diagram illustrates the flow of messages involved in a *B
 
 **Message Format**
 
+**Arbitrary events from the device**
+
 See [*Telemetry API*]({{< relref "Telemetry-API.md" >}}) for definition of message format. 
+
+## Well-known event message types
+
+Hono defines *well-known* events that are of a specific *content-type*.
+
+NB: currently there is only one such *well-known* event.
+
+### Lifecycle Notifications
+
+A specific event can signal that a device is `connected` (e.g. to signal that it is ready to receive a command for a specific time interval) 
+or that a device is `disconnected` (e.g to signal that it is not ready to receive a command anymore).
+ 
+*Business Applications* may want to react to such an event by sending a command upstream to the device.
+For that purpose, the protocol adapter instance that received such an event opens a receiver link for the device (scoped to its tenant) that
+is used to send the command to the appropriate protocol adapter instance.
+
+Such an event may be issued by the device itself or by a protocol adapter detecting that a device is connected or disconnected.
+ For further details on which part of a setup issues such an event see (... concept page).
+
+
+*Lifecycle notifications* may be decided to not being durable, but this is left to the setup of the *AMQP network* and is not further specified in the following.
```

Comment:
Are you saying that events are no longer *durable* by definition?

Reply:
No, where are you reading that?
I left it open if the connected/disconnected events are durable. Not by definition, just left open, see your comment to the discussion PR:

`This seems to be an issue worth discussing based on my comment above. We currently mark all events as durable btu I agree that we might want to relax this requirement and allow the issuer of the event to mark it as such or not.`

Would rephrase it that the sender can decide if the event should be durable.
Often IMHO the connected event may not make too much sense to be durable, though.

Reply:
No, I thought of changing that (in the spec and in the implementation).

But I would propose to do that later, and for now let the connection events be durable as all other events (better get something running before dealing with the optimizations).
So I will just delete the words about durability, then the default behaviour of events for now is valid here as well.